web/src/settings_org.js

```diff
@@ -262,36 +262,68 @@ function set_msg_move_limit_setting(property_name) {
     enable_or_disable_related_message_move_time_limit_setting(property_name, disable_setting);
 }
 
-function message_delete_limit_setting_enabled(setting_value) {
+function message_delete_limit_setting_enabled() {
```

Comment:
You're removing the `setting_value` param, so you'd need to remove its usages too. For example, [here](https://github.com/Vector73/zulip/blob/a43c0693b7ee065390f00612f9f685e899ef3959/web/src/settings_org.js#L279)

I wonder why the linter didn't complain..? Maybe cause it only does that detection on ts files?